resources/test/sse_data_schema.json

```diff
@@ -1873,6 +1874,17 @@
               "$ref": "#/definitions/U512"
             }
           ]
+        },
+        "new_validator": {
```

Comment:
This needs to be mentioned in the changelog. This is also backwards incompatible change as old clients cannot continue to parse this event without patching.

/cc @Fraser999

Reply:
The change has been noted here:
https://github.com/darthsiroftardis/casper-node/blob/Fix-redelegate/types/CHANGELOG.md#changed